package.json

```diff
@@ -42,7 +41,8 @@
   },
   "peerDependencies": {
     "react": "^0.14.0",
-    "react-dom": "^0.14.0"
+    "react-dom": "^0.14.0",
+    "react-tap-event-plugin": "^0.2.0"
```

Comment:
@zhiyelee I don't understand. Is `history` a dependency of `react-tap-event-plugin` I don't think so.